From 57bf275b260d74e9a1a6d84e180629b671e95fbe Mon Sep 17 00:00:00 2001
From: Pierre Schmitz <pierre@archlinux.de>
Date: Sun, 12 Jan 2014 11:31:05 +0100
Subject: [PATCH] Makefile.org: Fix usage of CC=gcc -m32

Commit 60adefa61025ffd7d56cf7ff8491008f783282bf introduced the use of the CC
variable without quoting. This breaks e.g. multilib builds where CC is set to
"gcc -m32".

Signed-off-by: Pierre Schmitz <pierre@archlinux.de>
---
 Makefile.org | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.org b/Makefile.org
index a9e2727..65a686d 100644
--- a/Makefile.org
+++ b/Makefile.org
@@ -302,7 +302,7 @@ libcrypto$(SHLIB_EXT): libcrypto.a fips_premain_dso$(EXE_EXT)
 			FIPSLD_CC="$(CC)"; CC=$(FIPSDIR)/bin/fipsld; \
 			export CC FIPSLD_CC FIPSLD_LIBCRYPTO; \
 		fi; \
-		$(MAKE) -e SHLIBDIRS=crypto  CC=$${CC:-$(CC)} build-shared; \
+		$(MAKE) -e SHLIBDIRS=crypto  CC="$${CC:-$(CC)}" build-shared; \
 		touch -c fips_premain_dso$(EXE_EXT); \
 	else \
 		echo "There's no support for shared libraries on this platform" >&2; \

-- 
1.8.5.2

